### PR TITLE
javascript moved to main.js

### PR DESCRIPTION
**Pull request by SelvinPL**
various fixes in editors

### CODE REVIEW

File: js/storage.js [hunk 1]
```diff
@@ -201,5 +204,6 @@ export function loadSingleUrl(url) {
 
 function postLoadUIUpdate() {
 	editors.setCurrentFile(Object.keys(files)[0]);
-	updateFileList();
+	main.updateFileList();
+	main.compileCode();
```
Comment (ISSOtm) [inline js/storage.js:208]: Why does this have to be added?

Comment (SelvinPL) [reply js/storage.js:208]: `main` ? or `compileCode` ?

Comment (SelvinPL) [reply js/storage.js:208]: `main` becuase updateFileList is in main module (created by moving all code from index.html to main.js)
`main.compileCode` i've added becuase i've noticed that code do not compile if loaded from url

Comment (ISSOtm) [reply js/storage.js:208]: The latter: why is this newly called? `main` has to be added due to the new scoping, that much I understand.

Comment (ISSOtm) [reply js/storage.js:208]: Okay. I'll split it into a separate commit, then, and make sure not to squash this PR.

**Pull request merged**